### ISSUE TITLE
Crash with Welcome to the Jungle

### ISSUE THREAD

**Issue by An-Sar** [WesCook/Nutrition#49]
It appears that a mob from Welcome to the Jungle eats chicken and possibly other foods, and is getting cast to EntityPlayer

i experienced this with a custom pack on a server and was able to recreate it in single player with only the two mods installed.

```
java.lang.ClassCastException: com.salvestrom.w2theJungle.mobs.entity.EntityLizardmanWitchDoctor cannot be cast to net.minecraft.entity.player.EntityPlayer
	at ca.wescook.nutrition.events.EventEatFood.checkFoodEaten(EventEatFood.java:93)
	at ca.wescook.nutrition.events.EventEatFood.finishUsingItem(EventEatFood.java:87)
```

Full Crash Log: https://paste.ee/p/Nj5V2

**Comment by WesCook** [comment]
Well that's an odd one.  Can't imagine why it'd be cast to an EntityPlayer.

I'll take a look.  Thanks for the (excellent) bug report!

**Comment by An-Sar** [comment]
yeah that was pretty weird to see. i tested a PrimalCore animal that also "eats" food items in world just to make sure, and that all worked out fine.

thank you for the excellent mod

**Comment by WesCook** [comment]
This has been fixed in [v2.8.0](https://github.com/WesCook/Nutrition/releases/tag/v2.8.0).  Thanks for the report!
